scripts/sct_utils.py

```diff
@@ -519,6 +519,44 @@ def printv(string, verbose=1, type='normal'):
         sys.exit(2)
 
 
+
+#=======================================================================================================================
+# send email
+#=======================================================================================================================
+def send_email(addr_to, addr_from='spinalcordtoolbox@gmail.com', subject='', message='', filename=None):
+    import smtplib
+    from email.MIMEMultipart import MIMEMultipart
+    from email.MIMEText import MIMEText
+    from email.MIMEBase import MIMEBase
+    from email import encoders
+
+    msg = MIMEMultipart()
+
+    msg['From'] = addr_from
+    msg['To'] = addr_to
+    msg['Subject'] = subject  # "SUBJECT OF THE EMAIL"
+    body = message  # "TEXT YOU WANT TO SEND"
+
+    msg.attach(MIMEText(body, 'plain'))
+
+    # filename = "NAME OF THE FILE WITH ITS EXTENSION"
+    if filename:
+        attachment = open(filename, "rb")
+        part = MIMEBase('application', 'octet-stream')
+        part.set_payload((attachment).read())
+        encoders.encode_base64(part)
+        part.add_header('Content-Disposition', "attachment; filename= %s" % filename)
+        msg.attach(part)
+
+    server = smtplib.SMTP('smtp.gmail.com', 587)
+    server.starttls()
+    server.login('spinalcordtoolbox@gmail.com', 'Awesome2014')
```

Comment:
Having the password in the code is a security risk. I suggest discussing a different strategy when returning from the holidays. In the meantime, change the password of the gmail account to avoid any possible hijacking of the email account.